variants/t1000-e/platformio.ini

```diff
@@ -34,6 +34,67 @@ build_src_filter = ${nrf52840_t1000e.build_src_filter}
 debug_tool = jlink
 upload_protocol = nrfutil
 
+[env:t1000e_repeater]
+extends = t1000-e
+build_flags = ${t1000-e.build_flags}
+  -I examples/companion_radio/ui-orig
+  -D ADVERT_NAME='"t1000-e Repeater"'
+  -D ADVERT_LAT=0.0
+  -D ADVERT_LON=0.0
+  -D ADMIN_PASSWORD='"password"'
+  -D MAX_NEIGHBOURS=8
+;  -D MESH_PACKET_LOGGING=1
+;  -D MESH_DEBUG=1
+  -D RX_BOOSTED_GAIN=true
+  -D RF_SWITCH_TABLE
+  -D DISPLAY_CLASS=NullDisplayDriver
```

Comment:
I don't think DISPLAY_CLASS is needed ... or do we have some use for the button in repeater ? or the LED ?

the remaining looks good to me (had made the same PR just after you)